bcda/main_test.go

```diff
@@ -328,6 +327,29 @@ func (s *MainTestSuite) TestCreateAlphaTokenCLI() {
 	l2 := strings.Split(anotherTokenInfo, "\n")
 	assert.NotEqual(l1[0], l2[0], "alpha expiration dates should be different (%s == %s)", l1[0], l2[0])
 	assert.NotEqual(l1[1], l2[1], "alpha token uuids should be different (%s == %s)", l1[1], l2[1])
+
+	// The following Okta tests should pass if you have OKTA_CLIENT_TOKEN set
```

Comment:
this can be uncommented if we're going to use it to test.  otherwise, let's delete so it doesn't confuse